modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -136,6 +136,20 @@ def test_read_multiple_csv_s3():
     df_equals(modin_df, pandas_df)
 
 
+@pytest.mark.parametrize(
+    "storage_options",
+    [{"anon": False}, {"anon": True}, {"key": "123", "secret": "123"}, None],
+)
```

Comment:
Is this just testing that the fact of passing a `storage_options` parameter doesn't trigger any error? If so, please leave an in-code comment about it. If it was intended to test that the passed parameters are actually applied, then I think we have to rewrite it into something more complicated (for example mock some s3 storage for this and check parameters that it receives).

Reply:
The comment with a note was added. More complex testing of `read_*` functions on work with s3 storages looks as a separate issue.